src/common/lib/oktaSignInWidgetHelper/oktaSignInWidgetHelper.js

```diff
@@ -17,35 +17,43 @@ export function showVerificationCodeField () {
   verificationCodeButtonLink?.click()
 }
 
-export function injectBackButton (functionCallback) {
+export function injectBackButton (context) {
+  // context will be the component instance that calls this function
+  if (!context || !context.onBackButtonClick) {
+    throw new Error('Context must be provided with a valid onBackButtonClick function.')
+  }
+
   const backButtonId = 'backButton'
   const backButtonText = 'Back'
   const buttonBar = document.querySelector('.o-form-button-bar')
   // Ensure the button is only added once
   if (buttonBar && !buttonBar.querySelector(`#${backButtonId}`)) {
     const backButton = angular.element(`<button id="${backButtonId}" class="btn btn-secondary" type="button">${backButtonText}</button>`)
-    // Add click behavior to go back a step
     backButton.on('click', (e) => {
       e.preventDefault()
-      this.$scope.$apply(() => functionCallback())
+      context.onBackButtonClick()
     })
     // Prepend the Back button before the "Next" button
     angular.element(buttonBar).prepend(backButton)
   }
 }
 
-export function initializeFloatingLabels () {
-    // Ensure floatingLabelAbortControllers is initialized
-  if (!this.floatingLabelAbortControllers) {
-    this.floatingLabelAbortControllers = [];
+export function initializeFloatingLabels (context) {
+  // context will be the component instance that calls this function
+  if (!context || !context.floatingLabelAbortControllers) {
+    throw new Error('Context must be provided with a valid floatingLabelAbortControllers array variable.')
   }
-  
+  // Ensure floatingLabelAbortControllers is initialized
+  if (!context.floatingLabelAbortControllers) {
+    context.floatingLabelAbortControllers = []
```

Comment:
I don't think this branch is reachable anymore because we'll throw an exception if `!context.floatingLabelAbortControllers`.